versionist.conf.js

```diff
@@ -1,4 +1,4 @@
-var _ = require('lodash');
+var _ = require(`${__dirname}/node_modules/lodash`);
```

Comment:
I suggest to make Versionist accept a function being returned from `versionist.conf.js`, and if it detects that, pass lodash directly to it, just for convenience. For example:

```js
// versionist.conf.js

module.exports = (utils) => {
    // utils.lodash.isEmpty()
    // utils.lodash.each()
    // etc ...
    return {
        // The usual configuration
    };
};
```